README.md

```diff
@@ -1,34 +1,51 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# A Next.js OAuth Agent for SPAs
 
-## Getting Started
+[![Quality](https://img.shields.io/badge/quality-test-yellow)](https://curity.io/resources/code-examples/status/)
+[![Availability](https://img.shields.io/badge/availability-source-blue)](https://curity.io/resources/code-examples/status/)
 
-First, run the development server:
+## Overview
 
-```bash
-npm run dev
-# or
-yarn dev
-```
+The OAuth Agent acts as a modern `Back End for Front End (BFF)` for Single Page Applications.\
+This implementation demonstrates the standard pattern for SPAs:
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+- Strongest browser security with only `SameSite=strict` cookies
+- The OpenID Connect flow uses Authorization Code Flow (PKCE) and a client secret
```

Comment:
PKCE != Authorization Code Flow
Suggest edit:
The OpenID Connect flow uses the Authorization Code Flow with [PKCE](https://curity.io/resources/learn/oauth-pkce/) and a client secret.

that link would be suitable as well: https://curity.io/resources/learn/pkce/